### DISCUSSION TITLE
Cannot knit Rmd document as 'Object 'nycflights' not found" although all code chunks works perfectly fine when executed

### DISCUSSION THREAD

**Discussion by princessblackswan** [teamalgoritma/community#771] [Q&A] [answered]
Hi all,

I'm working with the dataset called Nycflights (obtained after installing the nycflights13 package). It's also one of those preloaded packages from Tidyverse that you can access from data().

Basically, I'm done with the file and would like to knit it to html. All code chunks ran perfectly OK. However this error message comes up everytime I attempt to knit:

Quitting from lines 19-22 [unnamed-chunk-1] (LBB1.Rmd)
Error:
! object 'nycflights' not found
Backtrace:
 1. utils::str(nycflights)
Execution halted

I'm showing you the screenshot below. The library works perfectly fine, the dataset is loaded. What could have been the problem?

I have tried the following:
- Restarting R
- Restart and run code from the top (works fine)
- I have installed and also required the nycflights13 package
- I know probably not relevant here but I also made sure I've setwd to the right directory

Have you seen something like this before? Any suggestions?

<img width="642" alt="Screenshot 2023-09-29 at 20 45 42" src="https://github.com/teamalgoritma/community/assets/122248785/c43ce0a0-a42b-423f-8a11-fb2c1bb95014">

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear Ms. @princessblackswan 
The error may occur since you tried to call a function and provided undefined `nycflights` object.
Please refer to this documentation on how to call each of the tables this library provides.
https://github.com/tidyverse/nycflights13

I notice you are trying to access the flights information. It is provided in `flights` table. You should change `str(nycflights)` to `str(flights)` instead.

**Reply by princessblackswan** [reply]
Thank you Mba Saskia, changing variable name to `flights` works well for me.
